.github/workflows/reusable_benchmarks.yml

```diff
@@ -177,15 +181,42 @@ jobs:
             repo: context.repo.repo,
             body: body
           })
-  
-    - name: Upload HTML report
-      if: ${{ always() && inputs.upload_report }}
-      uses: actions/cache/save@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
-      with:
-        path: umf-repo/build/benchmark_results.html
-        key: benchmark-results-${{ github.run_id }}
 
-    - name: Get information about platform
-      if: ${{ always() }}
-      working-directory: ${{env.UMF_DIR}}
-      run: .github/scripts/get_system_info.sh
+    - name: Commit data.json and results directory
+      working-directory: results-repo
+      run: |
+        git config --global user.name "GitHub Actions Bot"
```

Comment:
Any chance of having this script as a part of the `benchmark-results` branch instead?

Reply:
That might be a good idea, let me think about it and I'll add this in a sep. PR